app/components/contributor-list/component.ts

```diff
@@ -0,0 +1,38 @@
+import { A } from '@ember/array';
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import { inject as service } from '@ember/service';
+
+export default class ContributorList extends Component.extend({
+    tagName: 'span',
+
+    contributors: A(),
```

Comment:
Might be reasonable to consider this a required argument/attribute, and just type it as an instance property without initializing. It would also be nice to establish a consistent way of documenting what's expected/available when using a component:
```ts
class ContributorList extends Component.extend({
    tagName: 'span',
}) {
    // Component parameters
    contributors: Array<Contributor>;

    // Class properties
    i18n = service('i18n');

    contributorList = ...
}
```

Reply:
I would disagree when it comes to arrays, specifically, because it's something that will cause a runtime error if the data hasn't been populated yet. If we're going to lazy load things, I'd err on the side of caution. For basic types, I would agree.